README.md

```diff
@@ -164,6 +175,17 @@ by executing `examples/kt/run_kt_experiment.ipynb` with appropriate parameters.
 ```
 See [examples/mmd_test/README.md](examples/mmd_test/README.md).
 
+#### [Debiased Distribution Compression](https://arxiv.org/pdf/2404.12290)
+```
+@article{li2024debiased,
```

Comment:
InProceedings{li2024debiased,
  title = 	 {Debiased Distribution Compression},
  author =       {Li, Lingxiao and Dwivedi, Raaz and Mackey, Lester},,
  booktitle = 	 {Proceedings of the 41st International Conference on Machine Learning},
  year = 	 {2024},
  volume = 	 {203},
  series = 	 {Proceedings of Machine Learning Research},
  month = 	 {21--27 Jul},
  publisher =    {PMLR},
}